src/main.rs

```diff
@@ -1649,6 +1650,66 @@ fn cmd_regenerate_audit_as(
     Ok(())
 }
 
+fn cmd_renew(out: &Arc<dyn Out>, cfg: &Config, sub_args: &RenewArgs) -> Result<(), miette::Report> {
+    trace!("renewing wildcard audits");
+    let mut store = Store::acquire_offline(cfg)?;
+    do_cmd_renew(out, cfg, &mut store, sub_args);
+    store.commit()?;
+    Ok(())
+}
+
+fn do_cmd_renew(out: &Arc<dyn Out>, cfg: &Config, store: &mut Store, sub_args: &RenewArgs) {
+    assert!(sub_args.expiring ^ sub_args.crate_name.is_some());
+
+    let new_end_date = cfg.today + chrono::Months::new(12);
+
+    let mut renewing: WildcardAuditRenewal;
+
+    if let Some(name) = &sub_args.crate_name {
+        match WildcardAuditRenewal::single_crate(name, store) {
+            Some(renewal) => {
+                renewing = renewal;
+                if renewing.is_empty() {
+                    info!("no wildcard audits for {name} are eligible for renewal (all have `renew = false`)");
+                    return;
+                }
+            }
+            None => {
+                warn!("ran `renew {name}`, but there are no wildcard audits for the crate");
+                return;
+            }
+        }
+    } else {
+        // Find and update all expiring crates.
+        assert!(sub_args.expiring);
+        renewing = WildcardAuditRenewal::expiring(cfg, store);
+
+        if renewing.is_empty() {
+            info!("no wildcard audits that are eligible for renewal have expired or are expiring in the next {WILDCARD_AUDIT_EXPIRATION_STRING}");
+            return;
+        }
+    }
+
+    renewing.renew(new_end_date);
+
+    write!(
+            out,
+            "Updated wildcard audits for the following crates and user ids to expire on {new_end_date}:"
+        );
```

Comment:
This formatting is weird.

Reply:
Agreed. It looks like for some reason cargo fmt doesn't want to reformat it. I know that can sometimes happen in macros but I'd expect `write!` to work.